es6-promises/index.js

```diff
@@ -0,0 +1,5 @@
+const takeAChance = require('./take-a-chance');
+
+const returnPromise = takeAChance('Laura');
+returnPromise.then(value => { console.log(value); });
+returnPromise.catch(error => { console.error(error); });
```

Comment:
If the `Promise` rejects, only the message should show, not the full `Error` object:

---

![image](https://user-images.githubusercontent.com/16143771/178056444-29172df9-b595-45c0-b7a6-27a8eda3e6be.png)

Reply:
<img width="726" alt="Screen Shot 2022-07-08 at 12 40 39 PM" src="https://user-images.githubusercontent.com/60913524/178059827-fa102712-1871-49b0-94c6-336e11eeb27d.png">